workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/MDFClaimForm/components/MDFClaimPage/schema/yup.ts

```diff
@@ -173,6 +176,28 @@ const claimSchema = object({
 										return true;
 									}
 								)
+								.test(
+									'templateValidation',
+									validateDocument.listOfLeadsDocuments
+										.templateContent.message,
+									async (listOfQualifiedLeadsFile) => {
+										if (
+											listOfQualifiedLeadsFile &&
+											!listOfQualifiedLeadsFile.documentId
+										) {
+											const result =
+												await partnerSpringBootOAuth2.handleFileUpload(
+													listOfQualifiedLeadsFile
+												);
+
+											return result.isValid
+												? true
+												: false;
```

Comment:
If the return of this `result.isValid` is a boolean, you don't need this ternary operator, since the return is the same as the one in the property.
```suggestion
											return result.isValid;
```